server/src/main/kotlin/org/example/myhome/client/DeviceKeyGenController.kt

```diff
@@ -31,7 +32,7 @@ class DeviceKeyGenController(
     @RequestBody body: String
   ): KeyGenResponse {
     log.debug("Key gen: $body")
-    val request = parseJson(body, KeyGenRequest::class.java)
+    val request: KeyGenRequest = objectMapper.readValue(body)
```

Comment:
I would suggest to avoid of using global variable (`objectMapper` in this case). better to rollback helper method to `JsonUtils` and add `private` modifier to `objectMapper`.

Reply:
I believe that we don't need these methods at all. why do you think that using objectMapper directly is bad idea?